targets/TARGET_TI/TARGET_MSP432/TARGET_MSP432P401R/TARGET_MSP432_LAUNCHPAD/BoardPinNames.h

```diff
@@ -0,0 +1,46 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2019 ARM Limited
```

Comment:
strictly speaking this should be only 2019 as all these are new files in this PR

Reply:
The MSP432 Launchpad has an on-board XDS110 debug probe and related USB connection. This is not compatible to DAPLink or jlink. I did not have the time yet to try out external DAPLink debuggers (the Lauchpad has a standard JTAG socket). This might work...